tests/integration-tests/tests/api/test_api/test_cluster_slurm/pcluster.config.update.yaml

```diff
@@ -1,75 +1,30 @@
 Image:
   Os: {{ os }}
-Tags:
```

Comment:
Why would the test be simplified?

Reply:
The Slurm test was unnecessarily complex for what we intended to test (I originally copied it from the configuration used for the update config tests), I reduced it to a more "basic" case that is sufficient for our purposes.